File: core/src/main/java/com/highpowerbear/hpbanalytics/database/ExchangeRateRepository.java
package com.highpowerbear.hpbanalytics.database;

import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

/**
 * Created by robertk on 4/13/2020.
 */
public interface ExchangeRateRepository extends JpaRepository<ExchangeRate, String> {

    List<ExchangeRate> findFirstByOrderByDateDesc();
}

File: core/src/main/java/com/highpowerbear/hpbanalytics/database/ExchangeRateRepository.kt
package com.highpowerbear.hpbanalytics.database

import org.springframework.data.jpa.repository.JpaRepository

/**
 * Created by robertk on 4/13/2020.
 */
interface ExchangeRateRepository : JpaRepository<ExchangeRate?, String?> {
    fun findFirstByOrderByDateDesc(): List<ExchangeRate>
}